controlplane/kubeadm/api/v1beta1/kubeadm_control_plane_types.go

```diff
@@ -117,6 +117,11 @@ type KubeadmControlPlaneSpec struct {
 	// The RemediationStrategy that controls how control plane machine remediation happens.
 	// +optional
 	RemediationStrategy *RemediationStrategy `json:"remediationStrategy,omitempty"`
+
+	// MachineNamingStrategy allows changing the naming pattern used when creating Machines.
+	// Note: InfraMachines & KubeadmConfigs will use the same name as the corresponding Machines.
```

Comment:
```suggestion
	// InfraMachines & KubeadmConfigs will use the same name as the corresponding Machines.
```

there is no need for the `Note:` here this is just another sentence providing the additional context.